### PR TITLE
Add official translations section under ReadMe 

### PR DESCRIPTION
**Pull request by taipeicityeth**
### Context 

Few communities have started translation work with different approaches, German team pivoted from branch to repository along with other translations (Japanese/Traditional Mandarin) thinking this might be the best way to scale. Expanded from this original [comment](https://github.com/pluralitybook/plurality/pull/6#issuecomment-1595052649):

The hope is we can 
- Standardize the translation workflow/structure to scale community contributions in decentralized ways while giving official recognition for legitimacy

Once there is a consensus on the best way between the initial communities, in the future we can 
- Make it simple for new languages to onboard with a suggested/tested framework to follow

With a similar structure of translated content for each repository, we could optimize to
- Automate updating the translation on the website directly from GitHub

### Description 

* This PR hopes to start the conversation between communities on if the above approach/direction makes sense 
* The PR adds a new section for recognized translations to demonstrate what the proposal here is suggesting

### CODE REVIEW

**Review by pluralitybook — CHANGES_REQUESTED** [review]
I support this conversation and leadership.  However, please correct the "liniks" to "links"

**Comment by giselechou** [comment]
Jacky has opened a place for discussion.Perhaps you two can start exchanging some ideas in here. @akinorioyama

**Comment by pluralitybook** [comment]
One comment on translations in general: while consistency with the root is important, I do not think we should overfocus on this especially at the microlinguistic level.  The book uses lots of double entendres and clever phrasings, and will have poetry.  Even some of the quotes are drawn from the US and Taiwanese context and there may be similar quotes anchored in other context...even data analysis focuses on a few countries and may be more relevant adapted to other linguistic contexts.  I would encourage experimentation along these lines, consistent with the underlying philosophy and arguments.

**Comment by akinorioyama** [comment]
> Jacky has opened a place for discussion.Perhaps you two can start exchanging some ideas in here. @akinorioyama

Thank you, @giselechou for introducing this thread of the pull request.
Hello @taipeicityeth . Where would you like to discuss how the initial merging process will be and how we can design the process for the near future and for a little more distant future?

I have outlined a few scenarios in very unfriendly graphics, mainly controlled by the functional flexibility of eleventy markdowns, eleventy plugins, and a web server, which may require different administrative processes to have the feature published to plurality.net.

Could we discuss the translation options on more flexible platforms such as online shared documents and brainstorming platforms? I would say we might have different priorities between the publishing platform and viewing platform, focusing primarily more on viewers of the Plurality book, and its "translations" (which includes displaying culturally differentiated content accordingly to the interests and the choices of the viewers.)

**Comment by taipeicityeth** [comment]
@pluralitybook, thanks - fixed the typo! 

Noted on not over-indexing on the structure and formatting consistency but allowing each sub-community/culture to flourish in its own way and be open to experimenting! 

I think the biggest blocker now would be how we can update our translated content on the website that is easy for translators and website maintainers. @pluralitybook could you help pull in the website maintainers here to discuss the best path forward? I imagine this is something we can quickly align on along with @akinorioyama and the German community 

@akinorioyama, let's discuss this here for transparency purposes to allow other folks to chime in - would love to see the graphs of yours! from what it sounds like, plurality.net is hosted on eleventy so we need to integrate the workflow between? Do you have admin access?   

Also, do you have a repository for where you are hosting the Japanese translation? [this one](https://github.com/akinorioyama/plurality) doesn't seem active

cc @GregCTWang who finished the Mandarin translation of chapter 2 for your thoughts too

**Comment by taipeicityeth** [comment]
Sorry @akinorioyama I missed your comment on discussing on a brainstorming doc elsewhere, that's totally fine! Do you have an existing document? if not, we can share one [I created here](https://taipeicityeth.notion.site/taipeicityeth/Plurality-Translation-Brainstorm-de7343d8f2da4f7ebdc689ca5548e3c8) or use your existing one:

**Comment by taipeicityeth** [comment]
cc @erichfi from the German side to join the conversation on our thoughts on creating a translation framework/system workflow different communities may leverage!

**Comment by taipeicityeth** [comment]
To make sure we're all talking about the same thing, based on my current understanding, we're hoping to figure out a system for the "workflow" section of the graph here. let me know if this graph is missing anything! cc @giselechou 

Screenshot 2023-06-26 at 3 24 04 PM

**Comment by akinorioyama** [comment]
@taipeicityeth 
Here are some graphics that I made for the basis for discussion. It is a little off from the standardized translation process for website localizations supported by services such as transifex (I think I have used several platforms, but don't recollect them).
As far as I can see now, there will be 3 major types of translations.
1. markdown display
2. eleventy-based display
3. different presentation mechanism such as manga or chat interactions (with trusted sites, most probably not hosted at plurality.net)

image

Let me answer the current setup in the message which includes your graphic for the workflow.

**Comment by akinorioyama** [comment]
> Workflow "How is this done currently?"

Technically speaking, the content at pluralitybook is read from plurality.net with some hardcoded URL destination of pluralitybook. On the plurality.net repo, I have adapted quickly to host the translated two chapters. On the pluralitybook repo, updates are done. I happened to have made changes to the plurality.net through the fork and pull request, but haven't been administering the repo (and CI/CD to the plurality.net site).
Here is a table that highlights the potential chnges and how it can be done in relation to the table shown in other message (https://github.com/pluralitybook/plurality/pull/30#issuecomment-1607103256), which includes the future plan to allow all the 3 types of translation mentioned in the message.
I planed to update the plurality.net site sections once discussion confirms the future design.

image

**Comment by taipeicityeth** [comment]
ah @akinorioyama I did not realize you were the maintainer of the plurality.net website, thanks for clarifying! 

This is definitely way more complex than I imagined - I'm afraid I don't fully comprehend the needs from a product perspective, so please help me understand! 

Is the plurality.net repo an open repo to look at? I'm curious how that is structured. 

Looking at the graph, it looks like from a product specs standpoint, we hope to support 
* Visualization 
* Structure 
* Interactions  
That might not be supported by markdown, but supported by eleventy for the most part except option 7?

Do you have an example of what these might look like to make sure we understand, and are these features needed already in the current Japanese translation?  How does an interaction look/behave? What type of visualization/structure are there that isn't supported by markdown? 

It looks like also from the URL reference under "Required Action" - we have the same idea of reading book content directly from GitHub? 

So to summarize a bit, seems like the biggest problem now is figuring out where to host the content, and under what format to satisfy all the requirements to display the book? 

sorry if I am asking dumb questions and lack understanding/context here, and thank you in advance for explaining!

**Comment by giselechou** [comment]
> 為了確保我們都在談論同一件事，根據我目前的理解，我們希望為此處圖表的“工作流程”部分找出一個系統。如果這張圖缺少任何東西，請告訴我！抄送@giselechou
> 
> 屏幕截圖 2023-06-26 下午 3 24 04

Hi Jacky, Aki and all, 
I've pasted below the content discussed in our last letter (au, Glen, me, Jacky and Greg) , specifically regarding systematization and Mandarin in Taiwan for everyone's reference.

From the letter and our previous discussions, I believe its quite clear. At the same time, we need to focus on expanding the "issues" （collaboration action）section.

In terms of community building, I believe we need to have a purpose to drive our actions. Therefore, my suggestion is to use "issues" as a clear goal and even take it a step further to cover a broader range. Below are two examples:

1. Once the Mandarin introduction version push, we should strive to gain more exposure for it. (e.g.for the community, can we push to increase the number of issues and links?)

2. Regarding other countries, perhaps translation hasn't started yet.
May different nations may have unique characteristics and might want to drive efforts in different ways. For example, Japan may prefer to use other methods such as doodling or related platforms.Therefore, for inclusivity, I still recommend developing "issues," which essentially means "collaboration." We can use this to drive efforts in different countries. (In terms of the system, I think we can integrate "issues" to help push the efforts. 

Directly publishing translated articles: Currently, there will be two Mandarin articles. These should be finalized in the next two weeks. The first one is already finalized, and we are just waiting for public.At the same time, suggested that we can integrate Japan, Germany, and other issues (for the repo, I'm thinking about the ability to integrate issues).

----
(for reference)
The following are my updates.

We've discussed this, and according to the current system structure, may develop a common system structure for multiple countries in the future. For example, Germany, Japan (thanks for your suggestion, Jacky. I know Akinori(Japan, akinori.oyama@gmail.com) , maybe you two could discuss more about this.

The aim is to encourage collaboration and the sharing of resources among different countries. Like the issues(Github) that can be further expanded, what is needed most are actual contributors to develop a Plural universe. (extended chapters).

Currently, I am handling specific chapters of the Mandarin translation. The approach is to align as closely as possible with the author's semantics, with au directly involved, and I mainly support au's needs. The Plurality articles (Audrey & Glen) that I am handling are deep and highly expandable, and the text of the articles is expected to inspire a high degree of collaboration.

In the future, there will still be many different chapters that need to be translated. Instead of developing 100 translations, perhaps the future approach should both focus on pursuing 1,000 expandability. Also,hope the two Mandarin chapters can go online sooner so that the community can start more swiftly.

Cheers,
Gisele

**Comment by akinorioyama** [comment]
> Workflow "What is the best format/structure to consume?"

I would say **the plurality.net can host what it can host, markdown and other eleventy-based formats**, but may be less prepared to host other formats that we can't predict all of the potential forms that may come in the future. I do already have different forms that I hope to have the content presented, but am thinking about making it first and host on other sites because the complexity and the required prediction power could be very high.
For that reason, I would say the current focus can be put only to on markdown format, then something eleventy can render with its already-available libraries (in the last iteration, I have added markdown-it-footnote and node-fetch).

Here are a few of the presentations (out of some others) I wish to implement in the near future. Some of them requir different mechanisms that enable other styles of visualizations.

**1. Gradual, non-linear,  need-based understanding** (https://twitter.com/LearningengAki/status/1653779003139063809)

<blockquote class="twitter-tweet" data-dnt="true"><p lang="en" dir="ltr">With further understanding of individual wishes, needs, or even pains, would it help guide how people like me can not only consume knowledge but also relate to both solutions &amp; problems that can be mapped out combined with discovered values through dialog. (w/d3sequences sunburst <a href="https://t.co/aDGaAGrtD0">https://t.co/aDGaAGrtD0</a> <a href="https://t.co/oYBnsdYsxK">pic.twitter.com/oYBnsdYsxK</a></p>&mdash; aki(codexDesignxVi(x̂i)xMEd student) = 0.000000001 (@LearningengAki) <a href="https://twitter.com/LearningengAki/status/1653779003139063809?ref_src=twsrc%5Etfw">May 3, 2023</a></blockquote>

**2. Modality (interest based)** (https://twitter.com/LearningengAki/status/1648003082339651590)
<blockquote class="twitter-tweet"><p lang="en" dir="ltr">Wishing to co-create a mechanism by which a proprietary system proposes different content types and structures with a deeper, clearer understanding the original message.<br>Pre-/Generated text/movie may focus on concerns/interests learned from screen input and open-ended questions. <a href="https://t.co/wVoTVYcL7p">https://t.co/wVoTVYcL7p</a> <a href="https://t.co/Zdb2lVaiin">pic.twitter.com/Zdb2lVaiin</a></p>&mdash; aki(codexDesignxVi(x̂i)xMEd student) = 0.000000001 (@LearningengAki) <a href="https://twitter.com/LearningengAki/status/1648003082339651590?ref_src=twsrc%5Etfw">April 17, 2023</a></blockquote>


**3. with differentiations prepared prior to use access/on-the-fly, componse texts with different focuses out from the content** without losing the context of the passage by providing sufficient background knowledge via supplemental texts created by the communities.
<blockquote class="twitter-tweet"><p lang="en" dir="ltr">Plurality is technology that recognizes, honors, and empowers cooperation across social and cultural differences by bridging through new voting and financing rules. It will only thrive to the extent we invest in it, but we&#39;re mistakenly fueling technology to be anti-democratic. <a href="https://t.co/MDByITn6zD">pic.twitter.com/MDByITn6zD</a></p>&mdash; aki(codexDesignxVi(x̂i)xMEd student) = 0.000000001 (@LearningengAki) <a href="https://twitter.com/LearningengAki/status/1647568682296672257?ref_src=twsrc%5Etfw">April 16, 2023</a></blockquote>

**4. in manga or dialog between famous characters to introduce the topic and discuss the benefits of the option.**
I am in discussion for the use of characters that recently open up an opportunity to allow the NFT holders to use characters under certain license agreements.

**Review by pluralitybook — COMMENTED** [review]
This looks good!  Time to get going.

**Comment by taipeicityeth** [comment]
> 

wow, these are really cool ideas! Can't wait to see how those turn out, especially the manga one, would be super helpful to help people understand the concepts in such a friendly way.

>  I would say the current focus can be put only to on markdown format

That sounds good, probably a good idea for us to take a small step at a time and focus on markdown first. I believe we should be ready to upload the Mandarin translation from @GregCTWang in markdown format. 

@GregCTWang, since we don't have visibility into your translation work yet - will need your input here from your formatting with markdown, do we want to upload it to a repository first and see if it can fit into @akinorioyama 's system to be updated on the website? What are your thoughts? 

@akinorioyama, what will we need in order to update the Mandarin translation on the website? What would be the easiest for you? Will it be a GitHub URL with different chapter folders? We might need your guidance here, thank you!

**Comment by akinorioyama** [comment]
May I limit my answer to for this reply to @giselechou ?
> @akinorioyama, what will we need in order to update the Mandarin translation on the website? What would be the easiest for you? Will it be a GitHub URL with different chapter folders? We might need your guidance here, thank you!

I hope this may clarify some of the questions that were raised by @taipeicityeth .

A language translation fits the option 1 in the table.
image

**In the pluralitybook repo, in the fork, md file** needs updated and the image files will need to be referred with absolute path (as the md file refers to relative path by default, but plurality.net site will try to display the images for the md file and in the process, it refers to the path, which, if relative, will not be displayed because the file doesn't reside within the plurality.net). Please take a look at the PR #1 to see the difference.

In the plurality.net repo, **1) modify one file** to set URL to the md file in pluralitybook repo, and **2) add one file** to relate the md file to the language and chapter where it should be displayed. The rough design plan is to display chaters and available languages.

**1) modify one file**
metadata of chapter 03 and github link (src/site/_data/github.js)

**2) add one file**
src/site/chapterversions/chapter_03/german.md
src/site/chapterversions/<chapter_number>/<language>.md (tentative naming convention)

The changes I made for the two new chapters are captured in PR 5 of plurality.net (https://github.com/alexrandaccio/plurality.net/pull/5/files).

Of course, language codes can be made to the ones like en_US, zh_TW, and ja_JP, and mapping data file can replace the mapping provided within the code now.

One of the deisns under rough design plans looks like this where the chapter names can be shown in only in number and available translations can show either the language name in its language or chapter name in its language.

image
